larky/src/main/java/com/verygood/security/larky/modules/vgs/aus/spi/AccountUpdaterService.java

```diff
@@ -0,0 +1,34 @@
+package com.verygood.security.larky.modules.vgs.aus.spi;
+
+import lombok.Builder;
+import lombok.Data;
+
+public interface AccountUpdaterService {
+  /**
+   * Get updated info for the provided card
+   *
+   * @param number card's number
+   * @param expireMonth card's expiration month
+   * @param expireYear card's expiration year
+   * @param name the name on the card
+   * @param clientId client id of service account to access calm API
+   * @param clientSecret client secret of service account to access calm API
+   * @return the updated card
+   */
+  Card lookupCard(
+      String number,
+      Integer expireMonth,
+      Integer expireYear,
+      String name,
+      String clientId,
+      String clientSecret);
+
+  @Data
+  @Builder
+  class Card {
```

Comment:
consider making `record` instead of `class`

Reply:
I tried to put record syntax here, but my IDE is showing red font. It seems like record is supported only in Java 14, are we using Java 14 in this larky project?

Reply:
You should be using JDK 17 here